tests/integration/abi_test.py

```diff
@@ -234,480 +228,3 @@ def test_roundtrip_abi_strategy(roundtrip_app):
     assert expected_mut == mut, inspector.report(
         args, "expected_mut v. mut", last_steps=last_rows
     )
-
-
-# ---- ABI Router Dry Run Testing - SETUP ---- #
```

Comment:
looks like these parts goes to https://github.com/algorand/pyteal/pull/634/files